### ISSUE TITLE
check_maintenance_incidents: consider falling back to source project for maintainers

### ISSUE THREAD

**Issue by jberry-suse** [openSUSE/openSUSE-release-tools#1380]
When a package is newly submitted (not target project nor `Factory`) then the maintainers are not detected as seen with [request#570561: discord -> Leap:15.0:NoneFree](https://build.opensuse.org/request/show/570561).

The conclusion is drawn from:

```
GET https://api.opensuse.org/search/owner?project=openSUSE%3ALeap%3A15.0%3ANonFree&binary=discord
GET https://api.opensuse.org/search/owner?binary=discord
[W] discord doesn't have maintainers
```

`osclib.core.owner_fallback()` does not have the source project available so either it would have to be provided as optional parameter or `check_maintenance_incidents` handle it. If changed to fallback it likely makes sense to review other calls to these methods to determine if/how it effects the other calls.

**Comment by jberry-suse** [comment]
Should likely be origin project fallback (especially when new project [not in target or Factory]).

Example: [#679962: KDE:Extra/plasma-pass -> openSUSE:Leap:15.1/plasma-pass](https://build.opensuse.org/request/show/679962)

**Comment by jberry-suse** [comment]
Fixed by #2283 with the introduction of `origin_maintainer_review_ensure()`.
